diff --git a/core/bootstrap/app.php b/core/bootstrap/app.php
index ee00e9409c2b3309fb8d0613b03216df6460e9b6..1edb12ea7a5b4e97daf197a51a17216e4d846f84 100644
--- a/core/bootstrap/app.php
+++ b/core/bootstrap/app.php
@@ -1,6 +1,7 @@
 <?php
 
 use App\Http\Middleware\AuthorizePermission;
+use App\Http\Middleware\UseDemoDatabase;
 use Illuminate\Foundation\Application;
 use Illuminate\Foundation\Configuration\Exceptions;
 use Illuminate\Foundation\Configuration\Middleware;
@@ -12,6 +13,10 @@
         health: '/up',
     )
     ->withMiddleware(function (Middleware $middleware): void {
+        $middleware->web(append: [
+            UseDemoDatabase::class,
+        ]);
+
         $middleware->alias([
             'core.permission' => AuthorizePermission::class,
         ]);
